.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.19-openshift-4.12
+  tag: rhel-8-release-golang-1.19-openshift-4.13
```

Comment:
This looks wrong. We do not use 4.13 imagesin 4.12. Did you target the right branch?